os-bpmsuite-executionserver/added/launch/bpmsuite-executionserver.sh

```diff
@@ -66,6 +78,29 @@ function configure_controller_access {
     fi
 }
 
+function configure_router_access {
+    local routerService="${KIE_SERVER_ROUTER_SERVICE}"
+    routerService=${routerService^^}
+    routerService=${routerService//-/_}
+    # host
+    local kieServerRouterHost="${KIE_SERVER_ROUTER_HOST}"
+    if [ "${kieServerRouterHost}" = "" ]; then
```

Comment:
According to my tests it works, and actually the comparison `[ "${var}" = "" ]` is used in many other cct scripts. Nevertheless `-z` is also widely used. Perhaps we could create a JIRA to unify the comparison in all cct scripts. Wdyt?